peer_mentoring/peer_mentoring/templates/allauth/account/login.html

```diff
@@ -1,4 +1,4 @@
-{% extends "account/base.html" %}
+{% extends "base.html" %}
```

Comment:
issue was that allauth templates were pointing to the wrong base template, TODO: fix all allauth templates @JAWD-001

Reply:
Okay, so change all the extends tag for the allauth templates then? I see, I hate that it was something so simple lol.

Reply:
Not easy to get to though LOL